lib/polyfill/media_capabilities.js

```diff
@@ -96,7 +96,7 @@ shaka.polyfill.MediaCapabilities = class {
       return res;
     }
 
-    if (mediaDecodingConfig.type == 'media-source') {
+    if (mediaDecodingConfig.type === 'media-source') {
```

Comment:
In all the code, we use == instead of ===, please revert this change.

Reply:
Done!